docs/automated_testing.md

```diff
@@ -460,7 +460,7 @@ Sometimes the characters in your code and the characters on screen aren't the sa
 <!-- Then the "a" link opens in the same window -->
 
 
-### Set fields
+### Set fields / Interact with page
```

Comment:
Hmm, this will break any links leading to this heading. Can you double check that there are none of those around that will need to be updated?

Also, can we think of other Steps we'll be creating that'll make an "interact with page" section good as its own section? Then we don't have to worry about it. I guess I don't love the idea of a multi-purpose heading.

Reply:
I checked when I changed it, there are no internal links to this header.

And IMO, "Interact with page" is the better label for the section, since whenever you set field, you're actually interacting with the page. It flows better with the previous header of "Observing things about the page" as well. `Set fields` is one of those things that is correct, but assumes a lot about the readers mental model of ALKiln, which they might not internalized yet.

If you agree with the name change, there are some other changes a bit unrelated, but would align better with this split (observing things about the page vs interacting with the page) that I could make; specifically, moving the upload and download files to observing and interacting with the page respectively.